lib/sms/Kconfig

```diff
@@ -6,9 +6,9 @@
 
 menuconfig SMS
 	bool "SMS subscriber"
-	select AT_CMD
-	select AT_CMD_PARSER
-	select AT_NOTIF
+	imply AT_CMD
```

Comment:
Why do you change this to imply?

Reply:
Otherwise I cannot have CONFIG_SMS=y in unity test prj.conf and at the same time have a CMock for at_cmd library. So this was the only way I could get it done properly without having to add all SMS library sources into Unity prj.conf. I'd like to have CONFIG_SMS=y in test prj.conf.

Reply:
ok, make sense